app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -396,6 +396,12 @@ class Config {
   bool githubPresubmitSupportedRepo(String repositoryName) {
     return supportedRepos.contains(repositoryName);
   }
+
+  Future<Map<String, dynamic>> get metricsCenterServiceAccountJson async {
+    final String rawValue =
+        await _getSingleValue('MetricsCenterServiceAccount');
```

Comment:
Is this a new service account or an existing one? Let's create a doc to document the origin, purpose and location. Going forward we'll need to track very closely all the service accounts used in flutter infrastructure.